qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/web/viewmodel/SearchCustomerViewModel.groovy

```diff
@@ -0,0 +1,23 @@
+package life.qbic.portal.qoffer2.web.viewmodel
+
+import life.qbic.datamodel.dtos.business.Customer
+
+/**
+ * A ViewModel holding data that is presented in a
+ * life.qbic.portal.qoffer2.web.views.SearchCustomerView
+ *
+ * This class holds all specific fields that are mutable in the view
+ * Whenever values change it should be reflected in the corresponding view. This class can be used
+ * for UI unit testing purposes.
+ *
+ *
+ * @since: 1.0.0
+ */
+class SearchCustomerViewModel {
+
+    List<Customer> customers = new ArrayList<>()
```

Comment:
I am not sure how explicit we want to name our variables that this is rather a suggestion than a request. Initially I was unsure about what the variables represent and just understood as I looked into other classes. Why not be explicit here (same for the following variables)?

```suggestion
    List<Customer> foundCustomers = new ArrayList<>()
```